Makefile

```diff
@@ -6,7 +6,7 @@
 #
 .PHONY: all doc clean test example
 CXX?=clang++
-CXXSTD?=c++14
+CXXSTD?=c++17
```

Comment:
as long as I would like to, I don't we can change it globally just like that, can we either make the plant_uml example c++17 only or make it c++14 compatible

Reply:
I'll look at doing that. Perhaps it can be feature flagged to use the old example in C++14 mode and this one in C++17.